### PR TITLE
[#298] Get rid of internal nexus

### LINKED ISSUE #298: Sort out nexus repositories usage
For now we store model images in 3 types of docker registry and have 4 different endpoints to access them:
1. Main Nexus repo for test-model images
2. Container with docker registry at localhost for unit tests
3. Per enclave Nexus docker registry available at localhost:port from docker socket at EDI (used for model images push)
4. Per enclave Nexus docker registry available at local url (used for model images labels inspection)

We should think about better implementation of model images store from build/deploy/test and code organization perspective.

### PR DESCRIPTION
**Pull request by mcdoker18**
This closes #298

### CODE REVIEW

**Comment by mcdoker18** [comment]
Deploy is okay https://cc.epm.kharlamov.biz/jenkins/job/Deploy_Legion_Release/241/
Please, review @akharlamov @kirillmakhonin

File: legion/tests/test_k8s.py [hunk 1]
```diff
@@ -163,6 +163,34 @@ def test_container_id_detection_docker(self):
             '9:devices:/docker/54d998b5f4232277ef245f7d93b0156dec3e149186916c557190983863bc7f57'
         ), '54d998b5f4232277ef245f7d93b0156dec3e149186916c557190983863bc7f57')
 
+    def test_parse_docker_image_url(self):
+        image_attributes = parse_docker_image_url(
+            'nexus.cc.epm.kharlamov.biz/legion-test-null-test-summation:1.0-190115092855.1.56ed5f4')
+        self.assertEqual(ImageAttributes(
+            host='nexus.cc.epm.kharlamov.biz', repo='legion-test-null-test-summation', ref='1.0-190115092855.1.56ed5f4'
```
Comment (kirillmakhonin) [inline legion/tests/test_k8s.py]: Could we remove this host names?

File: tests/robot/resources/browser.robot [hunk 2]
```diff
@@ -33,15 +33,3 @@ Login with dex
      Input Text 	                            xpath: ${DEX_AUTH_login_input} 	    ${STATIC_USER_EMAIL}
      Input Text 	                            xpath: ${DEX_AUTH_password_input} 	${STATIC_USER_PASS}
      Click Button                            xpath: ${DEX_AUTH_login_button}
-
```
Comment (kirillmakhonin) [inline tests/robot/resources/browser.robot]: Is this up-to date with develop?

File: tests/robot/tests/1_common/1.1_check_clusterwide.robot [hunk 3]
```diff
@@ -35,23 +32,6 @@ Checking if all replica sets, stateful sets, replication controllers are up and
...
 
-Check Nexus availability
-    [Documentation]  Check Nexus UI availability
-    [Tags]  nexus  ui  apps
-    Start browser    ${NEXUS_HOST}
-    Login with dex
-    Go To            ${NEXUS_HOST}/
-    Wait Nexus componens in menu
-
-Check Nexus Components available
```
Comment (kirillmakhonin) [inline tests/robot/tests/1_common/1.1_check_clusterwide.robot]: This too

File: tests/robot/resources/browser.robot (hunk 2 above)
Comment (mcdoker18) [reply tests/robot/resources/browser.robot]: Refresh PR.

File: tests/robot/tests/1_common/1.1_check_clusterwide.robot (hunk 3 above)
Comment (mcdoker18) [reply tests/robot/tests/1_common/1.1_check_clusterwide.robot]: Refresh PR.

File: legion/tests/test_k8s.py (hunk 1 above)
Comment (mcdoker18) [reply legion/tests/test_k8s.py]: Fixed

**Comment by mcdoker18** [comment]
bot, test all

**Comment by mcdoker18** [comment]
bot, test all

**Comment by mcdoker18** [comment]
bot, test all

**Comment by mcdoker18** [comment]
bot, test all

**Comment by mcdoker18** [comment]
bot, test all

**Comment by kirillmakhonin** [comment]
Please remove usage of external resources, keep only docker images store.

**Comment by mcdoker18** [comment]
bot, test all

**Comment by mcdoker18** [comment]
bot, test all

**Comment by mcdoker18** [comment]
bot, test all

**Comment by mcdoker18** [comment]
bot, test all

**Comment by mcdoker18** [comment]
bot, test all

**Comment by mcdoker18** [comment]
bot, test all

**Comment by mcdoker18** [comment]
bot, test all

**Comment by mcdoker18** [comment]
bot, test all

**Pull request merged**